compiler/optimizer/OMROptimizer.cpp

```diff
@@ -723,6 +723,7 @@ OMR::Optimizer::Optimizer(TR::Compilation *comp, TR::ResolvedMethodSymbol *metho
    // zero opts table
    memset(_opts, 0, sizeof(_opts));
 
+#if !defined(TR_OVERRIDE_OPTIMIZATION_INITIALIZATION)
```

Comment:
This needs to be addressed IMO - make it clear what this is and why one would use it.